Userland/Services/NotificationServer/ClientConnection.cpp

```diff
@@ -55,7 +55,7 @@ OwnPtr<Messages::NotificationServer::GreetResponse> ClientConnection::handle(con
 
 OwnPtr<Messages::NotificationServer::ShowNotificationResponse> ClientConnection::handle(const Messages::NotificationServer::ShowNotification& message)
 {
-    auto window = NotificationWindow::construct(message.text(), message.title(), message.icon());
+    auto window = NotificationWindow::construct(client_id(), message.text(), message.title(), message.icon());
```

Comment:
I like using `client_id()` as workaround!

Reply:
Nice side effect of using a single connection!